puppet/modules/freight/manifests/user.pp

```diff
@@ -5,6 +5,7 @@
   Stdlib::Absolutepath $stagedir,
   String $vhost,
   Variant[String, Array[String]] $cron_matches,
+  Optional[String[1]] $stable = undef,
```

Comment:
I moved this to https://github.com/theforeman/foreman-infra/pull/2120 as it's not directly related to this PR